continuous_integration/environment-3.8.yaml

```diff
@@ -5,7 +5,7 @@ channels:
   - nodefaults
 dependencies:
   # required dependencies
-  - python=3.8
+  - python=3.9
```

Comment:
We should either rename the file or drop it

Reply:
pandas 1.3 is tested in mindeps, same for numpy, so maybe worth dropping it completely?

Reply:
Yeah, that's a good point. Our convention of pinning to the lowest supported numpy / pandas version in our lowest supported Python build predates the mindeps builds. Especially now that we have the `mindeps-optional` build, it makes sense to have the version here be a little higher to improve numpy / pandas version coverage

Reply:
we're testing on:

| environment       | pandas version       |
|-------------------|----------------------|
| 3.11              | unrestricted (2.0.1) |
| 3.10              | 1.5.0                |
| 3.9               | 1.4.0                |
| mindeps-dataframe | 1.3.0                |
| mindeps-optional  | 1.3.0                |

we could change the py3.9 builds to all use 1.3.0 but then we're not testing on pandas 1.4 at all